.github/workflows/breaking-change-alert.yaml

```diff
@@ -0,0 +1,122 @@
+name: Breaking Change Notifications
+
+on:
+  workflow_call:
+    inputs:
+      sender_login:
+        description: 'Sender login'
+        required: true
+        type: string
+      sender_avatar:
+        description: 'Sender avatar URL'
+        required: true
+        type: string
+      repo:
+        description: 'Repository'
+        required: true
+        type: string
+      pr_number:
+        description: 'Pull request number'
+        required: true
+        type: number
+      pr_title:
+        description: 'Pull request title'
+        required: true
+        type: string
+      pr_body:
+        description: 'Pull request body'
+        required: false
+        default: '_Empty PR description_'
+        type: string
+      pr_base_ref:
+        description: 'Base branch of the pull request'
+        required: true
+        type: string
+      pr_author:
+        description: 'Pull request author'
+        required: true
+        type: string
+      event_action:
+        description: 'Event action'
+        required: true
+        type: string
+      pr_merged:
+        description: 'Pull request merged status'
+        required: true
+        type: boolean
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  notify-breaking-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Determine notification parameters
+        id: notification
+        run: |
+          if [[ "${{ inputs.event_action }}" == "closed" && "${{ inputs.pr_merged }}" == "true" ]]; then
+            echo "action=Merged" >> $GITHUB_OUTPUT
+            echo "color=#D00000" >> $GITHUB_OUTPUT
+            echo "icon=:rocket:" >> $GITHUB_OUTPUT
+          elif [[ "${{ inputs.event_action }}" == "closed" ]]; then
+            echo "action=Closed" >> $GITHUB_OUTPUT
+            echo "color=#1d9bd1" >> $GITHUB_OUTPUT
+            echo "icon=:information_source:" >> $GITHUB_OUTPUT
+          elif [[ "${{ inputs.event_action }}" == "reopened" ]]; then
+            echo "action=Reopened" >> $GITHUB_OUTPUT
+            echo "color=warning" >> $GITHUB_OUTPUT
+            echo "icon=:warning:" >> $GITHUB_OUTPUT
+          else
+            action=${{ inputs.event_action }}
```

Comment:
I think this needs to be changed/sanitized since it will be passed along to slack webhook.

Since it's the `else` case, maybe just something generic like `action=Changed`. Users can always go the PR for more details.